src/Components/DisplayOfArticles/ArticleList/index.jsx

```diff
@@ -0,0 +1,43 @@
+import "./ArticleList.css";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { getAllArticles } from "../../../utils/api";
+import { ArticleCard } from "./ArticleCard";
+
+export function ArticleList() {
+  const [newArticles, setNewArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getAllArticles()
+      .then((data) => {
+        setIsLoading(false);
+        setIsError(false);
+        setNewArticles(data.articles);
+      })
+      .catch(() => {
+        setIsLoading(false);
+        setIsError(true);
+      });
+  }, []);
+
+  function handleArticleToDisplay(article) {
```

Comment:
Does this need to be at this level, could it not be in the article card component